Sources/TokamakCore/Views/Image.swift

```diff
@@ -101,11 +101,16 @@ private class ResizableProvider: _AnyImageProviderBox {
 }
 
 public struct Image: _PrimitiveView, Equatable {
-  let provider: _AnyImageProviderBox
+  @_spi(TokamakCore)
+  public let provider: _AnyImageProviderBox
 
   @Environment(\.self)
   var environment
 
+  @_spi(TokamakCore)
+  @State
+  public var intrinsicSize: CGSize?
```

Comment:
shouldn't this property be underscored as SPI?

```suggestion
  public var _intrinsicSize: CGSize?
```